.circleci/config.yml

```diff
@@ -242,6 +285,8 @@ workflows:
       - request-preprod-approval:
           type: approval
           requires:
+            - cas1_e2e_tests
```

Comment:
I wonder if we should release this first without these requires so if there's a problem we don't block the pipeline. Then once it's green we follow-up with a small change to add them back in?

Reply:
Good plan. In a seperate push I commented out the requires and it seems to work, but that sounds sensible too